PeerioTechnologies/peerio-mobile
# Feat ui color scheme

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/4481/mobile-ui-colour-update
https://app.clubhouse.io/peerio/story/4487/mobile-update-icons-with-rebranding
https://app.clubhouse.io/peerio/story/4521/mobile-update-signup-on-boarding-badge-colours
https://github.com/PeerioTechnologies/peerio-icebear/pull/171
#### Testing instructions  
Dev: Ensure that nothing is broken
Design: Ensure that design is consistent with new UI branding (Reviewed couple of times by Saumya)

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
